### ISSUE TITLE
Settings derived from CONTENT_ROOT are inconsistent with assessment zip layout and settings module

### ISSUE THREAD

**Issue by benjaoming** [learningequality/ka-lite#4040]
Previously, one would set CONTENT_ROOT and other settings would be derived from it.

We need good copy-paste friendly code whereby the user can have the option to customize settings that previously were affected by CONTENT_ROOT.

This does not affect users still running on local_settings.py

Current docs only have this section:

> CONTENT_ROOT = “<path to desired content folder>” (default=ka-lite/content) This is the path that KA Lite will use to look for KA Lite video files to play. Change the path to another local directory to get video files from that directory. NB! Directory has to be writable for the user running the server in order to download videos.

**Comment by MCGallaspy** [comment]
The only two derived settings I could track down (in `kalite.settings.base`) are:
```python
KHAN_CONTENT_PATH = os.path.join(CONTENT_ROOT, "khan")
ASSESSMENT_ITEM_ROOT = os.path.join(CONTENT_ROOT, 'assessment')
```

Can we put these settings in an installed app? So that if a user creates a custom settings module, instead of setting `CONTENT_ROOT`, `KHAN_CONTENT_PATH`, AND `ASSESSMENT_ITEM_ROOT` separately, they can still just set `CONTENT_ROOT`? Or is this an anti-pattern?

**Comment by benjaoming** [comment]
@MCGallaspy yes, can't think of a better proposal... we've discussed something about assessment items having issues because 3 of the settings diverge from the layout of the current assessment items zip.

To summarize: The settings say that assessment_items.sqlite, assessment_items.versions, and assessment_items.json are located differently from where they are unzipped.

To fix this, we should probably move these specific files so they respect the settings.

Anyways, the point of this is also, that while fixing that issue, and because these settings are all derived from ASSESSMENT_ITEM_ROOT which is derived from CONTENT_ROOT, we should fix all of this together.

I don't mind assigning myself to this, because I'm packing up the .deb with assessment items bundled and I'm relying on this functionality to be fixed, anyways.

**Comment by MCGallaspy** [comment]
Okay, but I'm not sure if you're saying "yes" to this:

> Can we put these settings in an installed app?

or this:

> Or is this an anti-pattern?

If the former, then the documentation doesn't require much updating. If the latter, then we'll need to mention all three settings (though I think it's a bit overkill to have these all be separate settings).

**Comment by benjaoming** [comment]
I'm saying yes to allocating an app's settings to the app itself. I can make a suggestion for a refactor that extends towards a final goals.

#3757

**Comment by rtibbles** [comment]
@benjaoming What needs to be done on this still?

**Comment by rtibbles** [comment]
Assuming this was fixed in #4095. @benjaoming please reopen if not the case.
